addons/udes_stock_picking_batch/models/stock_picking_type.py

```diff
@@ -1,5 +1,17 @@
-from odoo import models
+from odoo import models, fields
 
 
 class StockPickingType(models.Model):
     _inherit = "stock.picking.type"
+
+    u_auto_assign_batch_pick = fields.Boolean(
+        string="Auto assign picks when added to a running batch",
+        help="Auto assign picks when added to a running batch",
+        default=False,
+    )
+
+    u_remove_unready_batch = fields.Boolean(
+        string="Auto remove unready picks from a running batch",
```

Comment:
Similar to above, maybe `Auto Remove Unready Batch Picks`?

Reply:
Done